pyodide-build/pyodide_build/serve.py

```diff
@@ -16,13 +16,13 @@ def end_headers(self):
 
 
 def make_parser(parser):
-    parser.description = "Start a server with the supplied " "build_dir and port."
+    parser.description = "Start a server with the supplied dist_dir and port."
     parser.add_argument(
-        "--build_dir",
+        "--dist_dir",
```

Comment:
Maybe let's allow both `build_dir` and `dist_dir` for backward compatibility?

Reply:
Yeah +1 for removing it in favor of `python -m http.server`.